src/hexo/themes/sonarwhal/layout/partials/home.hbs

```diff
@@ -12,53 +12,49 @@
             <div class="container home-container">
                 <section class="home-container--section">
                     <h1 class="headline">sonar</h1>
-                    <p class="title">a linting tool for the web</p>
+                    <p class="title">build better web sites&mdash;lint the web forward</p>
                     <div class="actions">
                         <a class="button button-cta button--red" href="/docs/user-guide/">Getting Started</a>
                         <a class="button button-cta button--red" href="/scanner/">Try it online</a>
                     </div>
                 </section>
-                <section class="home-container--section home-container__features">
-                    <h2 class="subheading subheading--home">Build better websites with sonar</h2>
+                <div class="home-container--section home-container__features">
                     <div class="layout layout--quarters">
                         <div class="module">
-                            <p class="subtitle--alt subtitle--home uppercase-text">Accessibility, security, interoperability and more!</p>
+                            <p class="subtitle--alt subtitle--home uppercase-text">Don’t burnout&hellip;</p>
                             <p class="feature_highlight">
-                                The web is complex. Knowing everything about web development and not making any mistakes can be challenging.
-                                sonar is meant to help developers regardless of their skill level, from the novice to the
-                                experienced one, to build websites and apps that work for everyone, are accessible, fast, secure, and
-                                take advantage of all the features the web has to offer.
+                                &hellip;trying to master all the things: sonar will help you ensure your code is accessible, fast, secure, 
```

Comment:
I think people could interpret this like "if I pass all the rules in sonar, my website is going to be fully accessible, fast, secure, etc." which is not the case. We will help you in those areas, but there are things we are not checking that could help in these and other areas.

What about something like:

> sonar will help you with your site's accessibility, speed, security, and more

Reply:
@rachelnabors any thoughts about my comment above?

Reply:
I have no concerns!